pine64/pinebook-pro/overlay.nix

```diff
@@ -0,0 +1,11 @@
+final: super:
+
+let
+  inherit (final) callPackage kernelPatches linuxPackagesFor;
+  kernels = callPackage ./kernel { };
+in
+{
+  pinebookpro-unstable-manjaro-kernel = kernels.manjaro-kernel;
+  pinebookpro-ap6256-firmware = callPackage ./firmware/ap6256-firmware { };
+  pinebookpro-keyboard-updater = callPackage ./keyboard-updater { };
```

Comment:
I don't think any of this needs to be an overlay. The module can just do `callPackage` directly. It's also easier to read because those packages don't just pop out of no where.
This can help for situations where people share a nixpkgs instance between different nixos machines and set: `nixpkgs.pkgs = nixpkgs;`

Reply:
I agree and I applied this change regarding the AP6256 firmware but I left the keyboard updater in the overlay since it's not directly used by anything in the configuration.